### PR TITLE
翻譯 io.js 3/20 週報

### PR DESCRIPTION
**Pull request by siygle**
先丟一個版本出來，還有一些不太確定怎麼翻的部分
需要其他大大們幫忙 XD #74

### CODE REVIEW

File: source/_drafts/weekly-update-march-20th.md [hunk 1]
```diff
@@ -3,46 +3,46 @@ date: 2015-03-20 09:02:55
...
+這週我們釋出兩個 io.js 版本，[v1.6.1](https://iojs.org/dist/v1.6.1/) 及 [v1.6.0](https://iojs.org/dist/v1.6.0/)，完整的更新項目可以在 [Github](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md) 上找到。
 
-## io.js 1.6 Release
+## io.js 1.6 釋出
 
 ### 1.6.1
 
-path: New type-checking on path.resolve() #1153 uncovered some edge-cases being relied upon in the wild, most notably path.dirname(undefined). Type-checking has been loosened for path.dirname(), path.basename(), and path.extname() (Colin Ihrig) #1216.
-querystring: Internal optimizations in querystring.parse() and querystring.stringify() #847 prevented Number literals from being properly converted via querystring.escape() #1208, exposing a blind-spot in the test suite. The bug and the tests have now been fixed (Jeremiah Senkpiel) #1213.
+* **path:** New type-checking on path.resolve() #1153 uncovered some edge-cases being relied upon in the wild, most notably path.dirname(undefined). Type-checking has been loosened for path.dirname(), path.basename(), and path.extname() (Colin Ihrig) #1216.
```
Comment (jihchi) [inline source/_drafts/weekly-update-march-20th.md:14]: **path:** `path.resolve()`新增型別檢查 [#1153](https://github.com/iojs/io.js/pull/1153)，不包含某些特殊案例，特別是 `path.dirname(undefined)`。已經放鬆 `path.dirname()`、`path.basename()` 以及 `path.extname()` 的型別檢查（Colin Ihrig）[#1216](https://github.com/iojs/io.js/pull/1216).

File: source/_drafts/weekly-update-march-20th.md [hunk 2]
```diff
@@ -3,46 +3,46 @@ date: 2015-03-20 09:02:55
...
-
-## Upcoming Events
-NodeConf tickets are on sale, June 8th and 9th at Oakland, CA and NodeConf Adventure for June 11th — 14th at Walker Creek Ranch, CA
-CascadiaJS tickets are on sale, July 8th — 10th at Washington State
-NodeConf EU tickets are on sale, September 6th — 9th at Waterford, Ireland
+* **node:** 一個新的 -r 或 — require 命令列參數可以用來在起始階段預先載入模組 (Ali Ijaz Sheikh) [#881](https://github.com/iojs/io.js/pull/881)。
+* **querystring:** parse() 及 stringify() 效能更快了 (Brian White) [#847](https://github.com/iojs/io.js/pull/847)。
+* **http:** http.ClientRequest#flush() 方法已被棄置，取而代之的是 http.ClientRequest#flushHeaders()，與 Node.js v0.12 的改變相同 [joyent/node#9048](https://github.com/joyent/node/pull/9048) (Yosuke Furukawa) [#1156](https://github.com/iojs/io.js/pull/1156)。
+* **net:** 允許 server.listen() 接受字串作為 port 的參數，例如 { port: “1234" } 以符合 [joyent/node#9268](https://github.com/joyent/node/pull/9268) 中 net.connect 接受相容格式的參數 (Ben Noordhuis) [#1116](https://github.com/iojs/io.js/pull/1116)。
+* **tls:** further work on the reported memory leak although there appears to be a minor leak remaining for the use-case in question, track progress at #1075.
```
Comment (jihchi) [inline source/_drafts/weekly-update-march-20th.md:23]: - **tls:** 回報的記憶體洩漏問題有更進一步的處理，儘管在這問題中顯示出這是個次要問題，可在 [#1075](https://github.com/iojs/io.js/issues/1075) 追蹤問題。

File: source/_drafts/weekly-update-march-20th.md [hunk 3]
```diff
@@ -3,46 +3,46 @@ date: 2015-03-20 09:02:55
...
+* **v8:** 修補一個整數溢位的問題，當 max_old_space_size 的數值大於 4096 的時候 (Ben Noordhuis) [#1166](https://github.com/iojs/io.js/pull/1166)。
+* **platforms:** io.js CI 回報已經通過 **FreeBSD** 以及 **SmartOS** (Solaris)。
+* **npm:** 升級至 2.7.1，細節請參閱 [npm CHANGELOG.md](https://github.com/npm/npm/blob/master/CHANGELOG.md#v271-2015-03-05)
+
+### 已回報問題
+
+* 可能存在與 TLS 相關的記憶體洩漏問題，細節在 [#1075](https://github.com/iojs/io.js/issues/1075)。
+* Surrogate pair in REPL can freeze terminal #690
+* io.js 無法編譯成靜態函式庫 [#686](https://github.com/iojs/io.js/issues/686)
+* process.send() is not synchronous as the docs suggest, a regression introduced in 1.0.2, see #760 and fix in #774
```
Comment (jihchi) [inline source/_drafts/weekly-update-march-20th.md:33]: `process.send()` 不是文件所建議的非同步，降級問題發生於 1.0.2 版，詳情請見 [#760](https://github.com/iojs/io.js/issues/760) 以及修正 [#774](https://github.com/iojs/io.js/issues/774)

**Comment by siygle** [comment]
@jihchi Thanks!!
